Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackIntegrationDriver.swift

```diff
@@ -7,19 +7,22 @@ import UIKit
  * An integration managing a workflow for end users to report feedback via Sentry.
  * - note: The default method to show the feedback form is via a floating widget placed in the bottom trailing corner of the screen. See the configuration classes for alternative options.
  */
-class SentryUserFeedbackIntegration {
+@objcMembers public class SentryUserFeedbackIntegrationDriver: NSObject {
```

Comment:
h: By making it public, this will be exposed to the user. Is that the intention? If so, how should the user interact with it directly?